### PR TITLE
Protect editing

### PR DESCRIPTION
**Pull request by edward**
Fixes https://github.com/Shopify/security-incidents/issues/27 by adding authentication around modification actions to places.

For review  @clayton-shopify

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,6 +1,2 @@
 class ApplicationController < ActionController::Base
-  protect_from_forgery with: :exception
```
Comment (clayton-shopify) [inline app/controllers/application_controller.rb]: I presume you didn't mean to remove CSRF protection.

File: test/controllers/places_controller_test.rb [hunk 2]
```diff
@@ -11,12 +11,21 @@ class PlacesControllerTest < ActionController::TestCase
     assert_not_nil assigns(:places)
   end
 
+  test "should require authentication to /new" do
```
Comment (clayton-shopify) [inline test/controllers/places_controller_test.rb]: It would be great to have similar tests for the other actions that require authentication.

Comment (edward) [inline test/controllers/places_controller_test.rb]: I’ll add those.

**Comment by edward** [comment]
@clayton-shopify okay – I think I’ve addressed the concerns you brought up. Let me know what you think.

**Comment by clayton-shopify** [comment]
LGTM, although I wasn't able to fully :tophat: because Google auth isn't working for me.

**Pull request merged**